packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -23,6 +23,13 @@ type BaseLinkCommonProps = {
   iconPosition?: 'left' | 'right';
   onClick?: (event: SyntheticEvent) => void;
   accessibilityLabel?: string;
+
+  /**
+   * Sets the size of the link
+   *
+   * @default medium
+   */
+  size?: 'small' | 'medium';
```

Comment:
add a test for size prop?

Reply:
Added in https://github.com/razorpay/blade/pull/761/commits/e35ad717133247c3e00611f6fb5a3623efdb3ed4